.lando.yml

```diff
@@ -29,13 +29,16 @@ services:
       - if [ ! -e $LANDO_MOUNT/wp-cli.yml ]; then cp $LANDO_MOUNT/config/wp-cli.yml $LANDO_MOUNT/wp-cli.yml; fi
     run:
       # Setup WordPress for contributing via git.
+      # TODO: Rename core-git to core or core-dev.
       - if [ ! -e $LANDO_MOUNT/public/core-git ]; then git clone --depth 1 https://github.com/WordPress/wordpress-develop.git $LANDO_MOUNT/public/core-git; fi
```

Comment:
In order for this to work properly, let's remove the `--depth 1` from here. Otherwise the clone will be shallow and only include the `master` branch, and checking out other branches will fail.

On my existing setup I fixed this by `git remote set-branches origin '*'` and `git fetch -v`, but all release branches should exist initially.